include/srtp.h

```diff
@@ -1424,7 +1408,7 @@ typedef enum {
 
 typedef struct srtp_event_data_t {
   srtp_t        session;  /**< The session in which the event happend. */
-  srtp_stream_t stream;   /**< The stream in which the event happend.  */
+  uintptr_t     ssrc;     /**< The ssrc in host order of the stream in which the event happend */
```

Comment:
Yeah I have no objections to the overall change - seems to make sense to me from a coding point of view at least, not too familiar with actually using the interface... 

Could leave the PR open a bit to see if others have inputs regarding the usability (terminology?), but pretty sure it is better to not have to depend on the private API in order to use the public API. :-)